src/BaGet/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -38,8 +43,11 @@ public static void AddBaGetContext(this IServiceCollection services)
                 var databaseOptions = provider.GetRequiredService<IOptions<BaGetOptions>>()
                     .Value
                     .Database;
-
-                options.UseSqlite(databaseOptions.ConnectionString);
+                if (string.IsNullOrEmpty(databaseOptions.ConnectionString))
+                {
+                    throw new InvalidOperationException($"The '{nameof(databaseOptions.ConnectionString)}' configuration is missing");
+                }
+                options.UseSqlite(databaseOptions.ConnectionString);//required Syntax: "Data Source=Filename"!!!

```

Comment:
>//required Syntax: "Data Source=Filename"!!! [](start = 68, length = 44)

The ADO.NET connection string format is well known, I think this comment is unneeded.